### DISCUSSION TITLE
A couple of issues with how things show on my Obsidian

### DISCUSSION THREAD

**Discussion by StavrosWTF** [BryanJenksCommunity/FAQ#95] [Q&A] [answered]
So hello. First of all, I would like to thank Bryan for all the great work he has done providing us with free knowledge. I am a total newb here so bear with me. I subscribed to the newsletter to get all the templates/css, follow the 4-hour long latest guide and implemented everything but now things don't look like Bryan's setup does.

As you can see from the screenshot, emojis next to names are missing. Any ideas on how to make those functional like Bryan's?

PS. I have installed every plugin he has recommended including most of core ones.
![Screenshot_1](https://user-images.githubusercontent.com/47122187/126873622-d0c3bfe7-7452-451b-835d-d6f20348588d.png)

**Comment by tallguyjenks** [comment, marked as answer]
oh yes, thats another CSS thing, there's a font to install if you want those to shot up for the file and folders, if not there's an alternative option

in the css check out item `2.2.2`

**Reply by StavrosWTF** [reply]
Oh I see, the css/ templates are older than your 2021. I will edit them ASAP. Thanks a lot man :)

**Reply by StavrosWTF** [reply]
Wanted to ask one more thing because I haven't really seen it in your video. Where do you notes go? Where are your feedlings, ferns, evergreen notees e.t.c?

Hopefully I am not asking a lot.

**Reply by tallguyjenks** [reply]
they're all top level at the root of my vault :)

**Reply by StavrosWTF** [reply]
Oh okay thanks a lot :) I thought you were just playing with kanban boards more (compared to your 2020 video).

**Reply by bluelephant825** [reply]
I tried installing Fira code and Hack Nerd fonts on my iMac running macOS Big Sur, but it didn't help! I also had a look at point 2.2.2 in Bryan's Gruvbox.css file but that didn't help either! I would very much appreciate some more detailed explanations to get this problem fixed. Thank you in advance.

**Reply by tallguyjenks** [reply]
@odebroqueville there's another font being used specifically for the folder icons called IcoMoon-free

![image](https://user-images.githubusercontent.com/29872822/134587209-4d29dbf7-6d28-43b6-a369-2ef1d0b4d118.png)

which is referenced in `2.2.2`

**Reply by bluelephant825** [reply]
@tallguyjenks Thank you, Bryan.

**Reply by wirikidor** [reply]
I just signed up for the newsletter today and started down my journey with Obsidian (abandoning Evernote).  I use a Windows desktop, a Macbook Pro laptop and an iPhone.

When I ran across this same problem I found the IcoMoon icons and installed them, but unless their font is part of an app in the AppStore  I couldn't find an easy way to accomplish this on the iPhone.

Instead I modified the CSS to use folder emojis just like the other folders and it looks quite nice.

```css
.nav-folder-children .nav-file-title-content:first-child::before {
  /* content: "\e924  ";
  font-family: "IcoMoon-Free"; */
  content: "📄 ";
  font-size:1.3em;
}
.nav-folder-children .nav-folder-title-content::before {
  /* content: "\e930  ";
  font-family: "IcoMoon-Free"; */
  content: "📂 ";
  font-size:1.3em;
}
```

The results
![obsidian](https://user-images.githubusercontent.com/8950557/137219459-f605f6c2-6b20-497a-9086-3db864950452.jpg)

**Reply by spandankeshav** [reply]
> @odebroqueville there's another font being used specifically for the folder icons called IcoMoon-free
> 
> ![image](https://user-images.githubusercontent.com/29872822/134587209-4d29dbf7-6d28-43b6-a369-2ef1d0b4d118.png)
> 
> which is referenced in `2.2.2`

Hi Bryan, I have downloaded the icon but could not do it. Do I need to change in css as well or just simply installing would solve this problem? Thanks 🙏

**Reply by tallguyjenks** [reply]
@spandankeshav installing the font should suffice

**Reply by spandankeshav** [reply]
Hi Bryan, how to install the font? Sorry for the dumb query but very beginner in obsidian 😀. Thanks a lot!

**Reply by elphin** [reply]
@spandankeshav 
You have to install the font on your machine. Go to https://icomoon.io/#icons-icomoon download the pack and find the font in the font folder. On windows you can simply double click the font file so it opens in a preview window, there you can click the 'Install' button.
